src/components/utils/hooks/useMessagePreviews.tsx

```diff
@@ -60,9 +60,6 @@ const useMessagePreviews = () => {
         newMessageProfiles.set(key, profile);
       }
       setMessageProfiles(newMessageProfiles);
-
-      // TODO: Add pagination of results once Lens fixes an issue with their API where pagination and cursor
```

Comment:
In chatting with Nick & Josh, this is running as intended so I removed the TODO.